lib/resources/r4/encounter.yaml

```diff
@@ -546,3 +546,92 @@ fields:
         system: http://terminology.hl7.org/CodeSystem/v3-ObservationValue
         info_link: http://hl7.org/fhir/r4/v3/ObservationValue/cs.html
     note: Encounters created with meta.security set with code 'UNCERTREL' will be returned with an external encounter type.
+
+- name: Client Organization Extension
```

Comment:
this has no binding and isn't needed

Reply:
https://github.com/cerner/fhir.cerner.com/pull/415/commits/346d1c84ec066ff89fd8f5dfdb7fec58712cc176#diff-876b1ef3d3fb228190dea594ca0a12abL550